src/stdatamodels/jwst/transforms/models.py

```diff
@@ -229,8 +229,10 @@ def __init__(self, prism_angle, name=None):
         self.outputs = ("n",)
 
     def evaluate(self, alpha_in, beta_in, alpha_out, prism_angle):
-        sangle = (math.sin(prism_angle))
-        cangle = (math.cos(prism_angle))
+        if isinstance(prism_angle, np.ndarray) and prism_angle.ndim == 1 and prism_angle.size == 1:
+            prism_angle = prism_angle[0]
+        sangle = np.sin(prism_angle)
+        cangle = np.cos(prism_angle)
```

Comment:
`prism_angle` is always a scalar.  astropy.modeling wraps parameters in arrays. Unless modeling changes, we could simply extract the value as in `prism_angle[0]` or `prism_angle.item()`. In any case there's no reason to use the numpy trig functions.

Reply:
Thanks! Using `item` seems cleaner and safer. I updated the PR based on your suggestion.